draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1028,11 +1028,16 @@ POST https://\<log server>/ct/v2/add-chain
 
 Inputs:
 
+: precertificate:
+  : The base64 encoded precertificate.  This field MUST be included if and only
+    if the certificate is being submitted as a precertificate.
+
 : chain:
-  : An array of base64 encoded certificates. The first element is the
-    certificate for which the submitter desires an SCT; the second certifies the
-    first and so on to the last, which either is, or is certified by, an
-    accepted trust anchor.
+  : An array of base64 encoded certificates. If the certificate is being
+    submitted as a precertificate, then the first element is the signer of the
+    precertificate; otherwise, the first element is the certificate being
+    submitted.  The second certifies the first and so on to the last, which
+    either is, or is certified by, an accepted trust anchor.
```

Comment:
"the certificate being submitted" doesn't make sense either, because it's actually multiple certificates (i.e., a certificate chain) that are being submitted.

I'd reword the description of the `chain` input as follows:
"An array of base64 encoded certificates. If the `precertificate` input is specified, then the first element is the signer of the precertificate; otherwise, the first element is the certificate for which the submitter desires an SCT. The second element certifies the first and so on to the last, which either is, or is certified by, an accepted trust anchor."